cmd/gaia/app/genesis.go

```diff
@@ -86,18 +75,15 @@ type GaiaGenTx struct {
 }
 
 // Generate a gaia genesis transaction with flags
-func GaiaAppGenTx(cdc *wire.Codec, pk crypto.PubKey) (
+func GaiaAppGenTx(cdc *wire.Codec, pk crypto.PubKey, genTxConfig gc.GenTxConfig) (
```

Comment:
in terms of namespacing, if we use `config` instead of gc we should also shorten the name of the types that are a part of config, aka rather than have `config.GenTxConfig` we should just have `config.GenTx`